### PR TITLE
Fixed issue 89: no matching function for call

### PR DESCRIPTION
**Pull request by softwaresale**
When building the project with networking, `item.toClipboard()`
is not passed with an action argument. I fixed it to do so.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -47,16 +47,16 @@ project(qlipper)
 
 set(QLIPPER_VERSION 5.1.1)
 
-option(ENABLE_NETWORK_CLIPBOARD_SHARING "Enables experimental network support for clipboard sharing" OFF)
+option(ENABLE_NETWORK_CLIPBOARD_SHARING "Enables experimental network support for clipboard sharing" ON)
```
Comment (palinek) [inline CMakeLists.txt]: Why did you change the default build option?

File: CMakeLists.txt [hunk 2]
```diff
@@ -47,16 +47,16 @@ project(qlipper)
...
 option(ENABLE_LXQT_AUTOSTART "Enables autostart for LXQt" OFF)
 option(USE_SYSTEM_QXT "Use system Qxt Library for global shortcuts; off for Qt5" OFF)
 
 
 LIST(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-
 find_package(Qt5Widgets "5.6" REQUIRED)
 find_package(Qt5LinguistTools REQUIRED)
+find_package(Qt5Network       REQUIRED)
```
Comment (palinek) [inline CMakeLists.txt]: This is only needed when the `ENABLE_NETWORK_CLIPBOARD_SHARING` is enabled...not?

File: CMakeLists.txt [hunk 3]
```diff
@@ -198,6 +198,7 @@ set_property(TARGET ${EXE_NAME} PROPERTY CXX_STANDARD_REQUIRED ON)
 
 target_link_libraries( ${EXE_NAME}
     Qt5::Widgets
+    Qt5::Network
```
Comment (palinek) [inline CMakeLists.txt]: Same as above...

**Comment by softwaresale** [comment]
My bad. I forgot to unset those values before I committed. Sorry

**Comment by palinek** [comment]
Please, squash the commits to leave one fixing the network issue and the other fixing build.
